app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -4,29 +4,23 @@
 
 package org.mozilla.fenix.home.recentbookmarks.view
 
-import androidx.compose.foundation.Image
-import androidx.compose.foundation.background
-import androidx.compose.foundation.clickable
-import androidx.compose.foundation.isSystemInDarkTheme
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.*
```

Comment:
We want to use single imports. I am not quite sure if this https://github.com/mozilla-mobile/android-components#style reinforces it.

Reply:
Caught again by Android Studio defaults. Updated